rts/lra/coordinator-quarkus/pom.xml

```diff
@@ -125,6 +130,35 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>native-docker</id>
```

Comment:
How about calling this "create-quarkus-native-image" or "create-quarkus-native-image-with-docker"

Reply:
I can rename both profiles if you prefer.

Reply:
I see, there is already a "native" - in that case I think the new name is fine for this PR. I will resolve this conversation.